go.mod

```diff
@@ -4,13 +4,18 @@ go 1.17
 
 require (
 	github.com/btcsuite/btcd v0.22.1
-	github.com/cosmos/cosmos-sdk v0.45.1
+	github.com/confio/ics23/go v0.7.0
+	github.com/cosmos/cosmos-sdk v0.45.4
+	github.com/cosmos/ibc-go/v2 v2.0.3
 	github.com/edgelesssys/ego v1.0.0
+	github.com/medibloc/panacea-core/v2 v2.0.4
```

Comment:
What about using the current master version of panacea-core?
```
go get github.com/medibloc/panacea-core/v2@master
```

Reply:
Thanks, I changed it at [60b9447](https://github.com/medibloc/panacea-doracle/pull/10/commits/60b94479345c6386c8704ce1d15fe3612605900e).